test/common/upstream/upstream_impl_test.cc

```diff
@@ -1429,6 +1429,30 @@ TEST_F(StaticClusterImplTest, MalformedHostIP) {
       "setting cluster type to 'STRICT_DNS' or 'LOGICAL_DNS'");
 }
 
+TEST_F(StaticClusterImplTest, NoHostsTest) {
```

Comment:
Can you add a comment above pointing out this is a regression and give the issue URL?

Reply:
Ok.